apps/deploy/app_config.py

```diff
@@ -0,0 +1,16 @@
+import os
+from django.apps import AppConfig
+import sys
+from django.conf import settings
+
+
+def _add_source_root(source_root):
+    sys.path.append(source_root)
+
+
+class HqDeploy(AppConfig):
+    name = "apps.deploy"
+
+    def ready(self):
+        # sneaky trick to add the HQ source onto the python path
+        _add_source_root(os.path.join(settings.HQ_SOURCE_ROOT))
```

Comment:
interesting so this is so we can use modules in HQ without actually loading HQ as a submodule. sounds reasonable to me. the reason we'd do this over a submodule is for simplicity and so we could possibly just point this to `/home/cchq/www/env/current` and thus make it faster/simpler to deploy chief?